homeassistant/components/ps4/__init__.py

```diff
@@ -124,3 +136,18 @@ def format_unique_id(creds, mac_address):
     """Use last 4 Chars of credential as suffix. Unique ID per PSN user."""
     suffix = creds[-4:]
     return "{}_{}".format(mac_address, suffix)
+
+
+async def async_service_handle(hass: HomeAssistantType):
```

Comment:
There's no await inside this function so it doesn't seem to need to be a coroutine.